pkg/graph/resolvers/plan_favorite.go

```diff
@@ -0,0 +1,66 @@
+package resolvers
+
+import (
+	"go.uber.org/zap"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/mint-app/pkg/authentication"
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/storage"
+)
+
+//PlanFavoriteCollection returns a list of favorites
+func PlanFavoriteCollection(logger *zap.Logger, principal authentication.Principal, store *storage.Store) ([]*models.PlanFavorite, error) {
+	favorites, err := store.PlanFavoriteCollectionByUser(logger, principal.ID(), false)
+	if err != nil {
+		return nil, err
+	}
+
+	return favorites, err
+}
```

Comment:
Do we still need this now that we don't have a query to get all of a users plan favorites? (Thinking this code can go away, and can be re-introduced if we find a need for it)

Reply:
Good call, I removed the store, and resolver methods for this, as well as the related SQL.